packages/coinjoin/src/backend/scanAccount.ts

```diff
@@ -118,7 +118,11 @@ export const scanAccount = async (
 
     let pending: Transaction[] = [];
     if (mempool) {
-        await mempool.update();
+        if (mempool.status === 'stopped') {
+            await mempool.start();
```

Comment:
when mempool starts does it does not update it, right? the `update` method is not run or is it updated other way?

Reply:
In this version incoming transactions are received only via websocket subscription, and `update` throws away the ones which shouldn't be in the mempool anymore. So the semantics behind this is:
- start listening to the mempool transactions right after the first account sync, and
- drop the invalid (mined or non-existent) mempool transactions after every consecutive sync